dev-lang/zig/zig-0.10.0.ebuild

```diff
@@ -75,7 +76,22 @@ src_configure() {
 
 src_test() {
 	cd "${BUILD_DIR}" || die
-	./zig2 build test -Dstatic-llvm=false -Denable-llvm=true -Dskip-non-native=true || die
+	local ZIG_TESTARGS=("-Dstatic-llvm=false -Denable-llvm=true -Dskip-non-native=true -Drelease -Dtarget=native")
+	./stage3/bin/zig build test-cases ${ZIG_TESTARGS[@]} || die
```

Comment:
Could we `edo` for these? (or `edob`) for improved logs? I'm going to merge anyway though.

Reply:
For some reason I thought they could be used only in eclasses... IMHO `edob` will be fine.

Reply:
Thanks! Please chuck that in a follow up (ok to just do it on next changes, no need to do a new PR just for this unless you want to).

Reply:
> Thanks! Please chuck that in a follow up (ok to just do it on next changes, no need to do a new PR just for this unless you want to).

OK! I'll do followup if upstream PR will be merged, and then — (even if PR will be rejected) I still should fix https://bugs.gentoo.org/851732 and https://bugs.gentoo.org/890234